libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -275,7 +281,10 @@ export function BmdPaperBallot({
 
   return withPrintTheme(
     <Ballot aria-hidden>
-      <Header>
+      <Header
+        className={largeTopMargin ? 'large-top-margin' : undefined}
```

Comment:
Any reason to use a class here vs just passing a prop to the `Header` styled component? (that's the pattern I usually see in our code for parameterized styling)

Reply:
Didn't know about conditional styling with the `styled-components` lib!